js/admin.vue

```diff
@@ -44,7 +45,16 @@ export default {
         }
     },
     ready() {
-        // Display an error identifier un uncaught error
+        // Displays a toast if read only mode is enabled
+        if (this.readOnlyEnabled) {
+            this.notifications.push({
+                type: 'error',
+                icon: 'exclamation-triangle',
+                title: this._('Attention'),
+                details: this._('Due to unusual activities, the creation of new content is currently disabled.'),
```

Comment:
```suggestion
                details: this._('Due to security reasons, the creation of new content is currently disabled.'),
```

Let's revert this, cf Mario's comment, my bad.